src/FlaUI.WebDriver.UITests/ElementTests.cs

```diff
@@ -168,14 +168,15 @@ public void GetElementRect_Default_IsSupported()
             using var driver = new RemoteWebDriver(WebDriverFixture.WebDriverUrl, driverOptions);
             var element = driver.FindElement(ExtendedBy.AccessibilityId("EditableCombo"));
 
+            var scaling = double.Parse(driver.FindElement(ExtendedBy.AccessibilityId("DpiScaling")).Text.ToString());
```

Comment:
Suggestion: Extract this to a helper method

Reply:
Yeah I would have done this but wasn't sure where to put it, given that the two places it's used are in different classes.

Reply:
Done.